app/forms/mappings/Mappings.scala

```diff
@@ -33,10 +33,31 @@ trait Mappings extends Formatters with Constraints {
                     nonNumericKey: String = "error.nonNumeric"): FieldMapping[Int] =
     of(intFormatter(requiredKey, wholeNumberKey, nonNumericKey))
 
+  protected def numeric(isCurrency: Boolean = false,
```

Comment:
this numeric method is not used anywhere

Reply:
I think this will be used in the currency entry fields